src/app/app.component.ts

```diff
@@ -21,14 +21,25 @@ export class AppComponent implements OnInit {
   public title = 'virtrolio-site';
   public showCookieAlert = true;
   private showCookieAlertValue: string;
+  public showChangelog = true;
+  public showChangelogValue: string;
 
   constructor(private cookieService: CookieService) {
   }
 
-  /** Dismisses cookie alert when X is clicked. Sets cookie to 'false'. */
-  dismissCookieAlert() {
-    this.cookieService.set('new-user-cookie', 'false', 365);
-    this.showCookieAlert = false;
+  /**
+   * Changes a cookie to 'false' and changes the corresponding cookie value variable to false as well
+   * @param cookie - cookie to be set to false
+   */
+  falsifyCookie(cookie: string) {
```

Comment:
This function seems to be a bit misleading. From the documentation and the name of the function, it would appear that if you pass any string, it will set that cookie to false. However, it only works for `new-user-cookie` and `changelog-cookie`, doing nothing for other cookies. Maybe add support if another cookie is passed?

Reply:
^

Reply:
I'm gonna super simplify the cookie code, just gonna put the code into the HTML `*ngIf`.